code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -76,6 +76,52 @@
 		RND_CATEGORY_MECHFAB_CYBORG + RND_SUBCATEGORY_MECHFAB_CYBORG_CHASSIS
 	)
 
+//Advanced Robotic Limbs
+
+/datum/design/advanced_l_arm
+	name = "Advanced Left Arm"
+	id = "advanced_l_arm"
+	build_type = MECHFAB
+	build_path = /obj/item/bodypart/arm/left/robot/advanced
+	materials = list(/datum/material/iron=SHEET_MATERIAL_AMOUNT*10, /datum/material/titanium=SHEET_MATERIAL_AMOUNT*3, /datum/material/gold=SHEET_MATERIAL_AMOUNT*3)
```

Comment:
```suggestion
	materials = list(
		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 10,
		/datum/material/titanium = SHEET_MATERIAL_AMOUNT * 3,
		/datum/material/gold = SHEET_MATERIAL_AMOUNT * 3,
	)
```
I know every other design doesn't multi-line this but it's just so much easier to read at a glance

Reply:
cleaned up the whole thing for ya